src/views/home/components/token-list/token-list.styles.ts

```diff
@@ -14,19 +14,19 @@ const useListStyles = createUseStyles((theme: Theme) => ({
   },
   card: {
     width: "100%",
-    maxWidth: 426,
-    padding: theme.spacing(2),
-  },
-  search: {
+    maxWidth: 500,
```

Comment:
Why extend it to 500?

Reply:
To make sure all addresses fit and give some extra room till the `x` blue cross:

![long-address](https://user-images.githubusercontent.com/15896177/169350015-81fcc661-db3b-44f3-b8e2-b88b138dd4ca.gif)